test/Altinn.AccessManagement.Tests/Controllers/DelegationsControllerTest.cs

```diff
@@ -2362,6 +2333,280 @@ public async Task MaskinportenDelegation_ValidationProblemDetails_TooLowAuthenti
+        {
+            // Arrange
+            string toParty = "50004221";
+            _client = GetTestClient(httpContextAccessor: GetHttpContextAccessorMock("party", toParty));
+            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", PrincipalUtil.GetToken(20001337, 50001337));
+            _client.DefaultRequestHeaders.Add("party-organizationumber", "810418532");
+
+            StreamContent requestContent = GetRequestContent("RevokeReceivedMaskinportenScopeDelegation", "jks_audi_etron_gt", $"p50005545", $"p{toParty}");
+
+            // Act
+            HttpResponseMessage response = await _client.PostAsync($"accessmanagement/api/v1/organization/delegations/maskinportenschema/received/revoke", requestContent);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.NoContent, response.StatusCode);
+        }
+
+        /// <summary>
+        /// Test case: RevokeReceivedMaskinportenScopeDelegation performed by authenticated user 20001337 for the reportee organization 910459880 of the jks_audi_etron_gt maskinporten schema resource from the resource registry,
+        ///            which have received delegation from the party 50005545
+        ///            In this case:
+        ///            - The user 20000490 is DAGL for the From unit 910459880
+        /// Expected: RevokeReceivedMaskinportenScopeDelegation returns 204 No Content
+        /// </summary>
+        [Fact]
+        public async Task RevokeReceivedMaskinportenScopeDelegation_DAGL_FromOrgNo_Success()
+        {
+            // Arrange
+            string toParty = "50004221";
+            _client = GetTestClient(httpContextAccessor: GetHttpContextAccessorMock("party", toParty));
+            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", PrincipalUtil.GetToken(20001337, 50001337));
+            _client.DefaultRequestHeaders.Add("party-organizationumber", "810418532");
+
+            StreamContent requestContent = GetRequestContent("RevokeReceivedMaskinportenScopeDelegation", "jks_audi_etron_gt", $"910459880", $"810418532");
+
+            // Act
+            HttpResponseMessage response = await _client.PostAsync($"accessmanagement/api/v1/organization/delegations/maskinportenschema/received/revoke", requestContent);
+
+            // Assert
+            Assert.Equal(HttpStatusCode.NoContent, response.StatusCode);
+        }
+
+        /// <summary>
+        /// Test case: RevokeReceivedMaskinportenScopeDelegation performed by authenticated user 20000095 for the reportee party 50004221 of the jks_audi_etron_gt maskinporten schema resource from the resource registry,
+        ///            which have received delegation from the party 50005545
+        ///            In this case:
+        ///            - The user 20000095 is ADMAI (Tilgangsstyrer) for the To party 50004221
+        /// Expected: RevokeReceivedMaskinportenScopeDelegation returns 204 No Content
```

Comment:
Feil excected

Reply:
Fixed for both this and RevokeReceivedMaskinportenScopeDelegation_ValidationProblemDetails_InvalidFrom case below